### PR TITLE
Smart contracts have only one termination function

### LINKED ISSUE #34: Problem: smart contracts have 3 termination functions
Problem: Our smart contracts have 3 functions that achieve the same functionality: 
- `setClaimable`
- `cancelOpenStream`
- `terminate`

Solution: remove redundant functions and refactor contracts to ensure only `terminate` function remains.

### PR DESCRIPTION
**Pull request by accompany1205**
resolves #34

### CODE REVIEW

File: contracts/StreamManager.sol [hunk 1]
```diff
@@ -135,31 +129,17 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
             _rate,
             _terminationPeriod,
             _cliffPeriod,
-            block.timestamp, // created at
```
Comment (markholdex) [inline contracts/StreamManager.sol:138]: @accompany1205 why did you remove created at? because the struct still has it?

**Review by markholdex — CHANGES_REQUESTED** [review]

File: contracts/StreamManager.sol (hunk 1 above)
Comment (accompany1205) [reply contracts/StreamManager.sol:138]: Where's it?

Comment (accompany1205) [reply contracts/StreamManager.sol:138]: We don't need createdAt for now. If we need it in the future, I will add.

**Review by markholdex — APPROVED** [review]
